src/helpers/LinkButtonComponent.types.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * The proptypes for {@link LinkButtonComponent}
+ */
+export interface LinkButtonComponentProps {
+  id?: string;
+  className?: string;
+  isStartButton?: boolean;
+  href: string;
+  children: React.ReactNode;
+}
+
+/**
+ * The type of a valid link button component to be passed to {@link ComponentRegister}.
```

Comment:
```suggestion
 * The type of a valid link component to be passed to {@link ComponentRegister} to be used for {@link LinkButton}.
```

Unlike `ComponentRegister.Link` this one has a specific use case.